Lib/test/test_csv.py

```diff
@@ -1560,6 +1560,20 @@ def test_ordered_dict_reader(self):
              OrderedDict([('fname', 'John'), ('lname', 'Cleese')]),
             ])
 
+class HugeDataTest(unittest.TestCase):
+    def test_write_huge_data(self):
```

Comment:
This test should probably be guarded with `support.requires("largefile", ...)` so that it only runs when the resource is enabled via `python -m test -u largefile`

Reply:
I would prefer not to test this, 2GB files are too much for this kind of change.

Reply:
Not disagreeing, but would you mind elaborating on why we shouldn't test this, just for future reference? :slightly_smiling_face:

Reply:
Tests are rather hard to reason about. I would put it this way: since it is a very slow and heavy test that tests an integer overflow. It is not specific to any arch / os / runtime / etc.

Reply:
Makes sense, thanks for the explanation :)